src/lib.rs

```diff
@@ -98,6 +99,26 @@ impl PGAppend<*mut i8> for pg::StringInfo {
     }
 }
 
+struct Wrapped(pg::Enum_ReorderBufferChangeType);
+
+impl fmt::Display for Wrapped {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        use pg::Enum_ReorderBufferChangeType::*;
+        let formatted_token = match self.0 {
+            REORDER_BUFFER_CHANGE_INSERT => "insert",
+            REORDER_BUFFER_CHANGE_UPDATE => "update",
+            REORDER_BUFFER_CHANGE_DELETE => "delete",
+            REORDER_BUFFER_CHANGE_INTERNAL_SNAPSHOT => "internal_snapshot",
+            REORDER_BUFFER_CHANGE_INTERNAL_COMMAND_ID => "internal_command_id",
+            REORDER_BUFFER_CHANGE_INTERNAL_TUPLECID => "internal_tuplecid",
+            REORDER_BUFFER_CHANGE_INTERNAL_SPEC_INSERT => "internal_spec_insert",
+            REORDER_BUFFER_CHANGE_INTERNAL_SPEC_CONFIRM => "internal_spec_confirm",
+            REORDER_BUFFER_CHANGE_MESSAGE => "change_message"
```

Comment:
So it is a catch all on 9.5, and 9.6 (not on 9.4).

On 9.5/9.6 Rust evaluates this as a `_` (since the value doesn't exist). However, it will also generate warnings that this code is unreachable, because we manually specify every single value. So on the places where it acts as a catch all it's impossible to hit.

However on 9.4 this generates all possible values for the enum. If this weren't here it would fail to build on 9.4 only since it didn't cover all the options.

If this were an underscore we would lose the ability to print out which change it was later on in the file (where we print unrecognized change action).